flight_phase_files/HALO_RF03_20200124_info.yaml

```
@@ -19,59 +19,115 @@ remarks:
 - Circle pattern, in the end excursion towards NTAS.
 - Deeper convection in the SE circle, spreading cold pools and stratiform clouds.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-01-24 09:50:17
-  end: 2020-01-24 10:48:51
-- kind: circle_break
-  name: cb1to2
-  start: 2020-01-24 10:48:51
-  end: 2020-01-24 11:00:08
-- kind: circle
-  name: c2
-  start: 2020-01-24 11:00:08
-  end: 2020-01-24 12:00:00
-- kind: circle_break
-  name: cb2to3
-  start: 2020-01-24 12:00:00
-  end: 2020-01-24 12:15:10
-- kind: circle
-  name: c3
-  start: 2020-01-24 12:15:10
-  end: 2020-01-24 13:10:03
-- kind: circle_break
-  name: cb3to4
-  start: 2020-01-24 13:10:03
-  end: 2020-01-24 13:30:05
-- kind: circle
-  name: c4
-  start: 2020-01-24 13:30:05
-  end: 2020-01-24 14:25:07
-- kind: circle_break
-  name: cb4to5
-  start: 2020-01-24 14:25:07
-  end: 2020-01-24 14:40:08
-- kind: circle
-  name: c5
-  start: 2020-01-24 14:40:08
-  end: 2020-01-24 15:35:05
-- kind: circle_break
-  name: cb5to6
-  start: 2020-01-24 15:35:05
-  end: 2020-01-24 15:50:06
-- kind: circle
-  name: c6
-  start: 2020-01-24 15:50:06
-  end: 2020-01-24 16:45:10
-- kind: straight_leg
-  name: leg1_outbound_NTAS
+- kinds:
+  - circle
+  name: Circle-1
+  irregularities: null
```

I thought that we use empty lists if there are no irregularities to report. I will exchange all instances in the file and in case I am wrong, simply don't accept the review ;-)  
Actually I would add here a comment on the 2 sondes without launch detect.
```suggestion
  irregularities: ['2 sondes with no launch detect']
```